app/src/main/java/org/mozilla/reference/browser/EngineProvider.kt

```diff
@@ -16,10 +17,27 @@ import mozilla.components.lib.crash.handler.CrashHandlerService
 import org.mozilla.geckoview.GeckoRuntime
 import org.mozilla.geckoview.GeckoRuntimeSettings
 import org.mozilla.reference.browser.ext.isCrashReportActive
+import java.io.File
 
 object EngineProvider {
 
     private var runtime: GeckoRuntime? = null
+    private var cliqz: CliqzExtensionFeature? = null
+
+    private const val geckoViewConfigPath = "geckoview-config.yaml"
+
+    /**
+     * Import a geckoview config YAML file from assets and return a File that the GeckoRuntime
+     * can load it from.
+     */
+    private fun importGeckoConfig(context: Context): File {
+        val configAssets = context.assets.open(geckoViewConfigPath)
+        val configFile = File(context.cacheDir, geckoViewConfigPath)
+        configFile.createNewFile()
+        configAssets.copyTo(configFile.outputStream())
+        configAssets.close()
+        return configFile
```

Comment:
Will be better to copy the file only if it doesn't exist already, possibly use an `AtomicFile`.